src/test/java/org/example/SimpleMathLibraryTest.java

```diff
@@ -0,0 +1,35 @@
+package org.example;
+
+
+import org.junit.Test;
+import java.util.logging.Logger;
+
+
+public class SimpleMathLibraryTest  {
+    private static Logger LOGGER = Logger.getLogger(SimpleMathLibrary.class.getName());
+    private SimpleMathLibrary mathLibrary;
+
+    public SimpleMathLibraryTest() {
+        this.mathLibrary = new SimpleMathLibrary();
+    }
+
+    @Test
+    public void testAdd() {
+        double result = mathLibrary.add(2.5, 3.5);
+        if (result == 6.0) {
+            LOGGER.info("OK");
+        } else {
+            LOGGER.info("NOK");
+        }

```

Comment:
Let's make it closer to real-world test cases. Get rid of if-else and simply check whether the value is the one you expect or not. Use JUnit assertions `assertEquals` etc.